sass/layout/_sidebar-left.scss

```diff
@@ -599,26 +599,24 @@
         .SidebarMenu:not(.menuOpen) {
             display: none;
         }
+    }
 
-        &.draggingOverHeader:hover {
-            //background: radial-gradient(60.83% 3421.87% at 50% 50%, rgba(255, 255, 255, 0.4) 0%, rgba(255, 255, 255, 0) 100%);
-            box-shadow: inset -1px 0 0 rgba(var(--sidebar-text-rgb), 0.60), inset 0 -1px 0 rgba(var(--sidebar-text-rgb), 0.6), inset 1px 0 0 rgba(var(--sidebar-text-rgb), 0.6), inset 0 1px 0 rgba(var(--sidebar-text-rgb), 0.6);
-            border-radius: 4px;
+    .SidebarChannelGroup:not(.dmCategory) .draggingOver .SidebarChannelGroupHeader:hover {
+        box-shadow: inset -1px 0 0 rgba(var(--sidebar-text-rgb), 0.60), inset 0 -1px 0 rgba(var(--sidebar-text-rgb), 0.6), inset 1px 0 0 rgba(var(--sidebar-text-rgb), 0.6), inset 0 1px 0 rgba(var(--sidebar-text-rgb), 0.6);
+        border-radius: 4px;
 
-            .SidebarChannelGroupHeader_groupButton {
-                background-color: inherit;
-            }
+        .SidebarChannelGroupHeader_groupButton {
+            background-color: inherit;
         }
-
     }
+
     .SidebarChannelGroup .SidebarChannelGroupHeader_groupButton {
         height: 32px;
         padding: 0;
         font-size: 12px;
         font-family: 'Open Sans', sans-serif;
         font-weight: 600;
         text-transform: uppercase;
-        overflow: hidden;
         text-overflow: ellipsis;
```

Comment:
If you are removing `overflow: hidden;` here, the corresponding `text-overflow: ellipsis` style will have no effect anymore so you should probably remove it as well. Neither appears to be necessary for truncating text here anyways as the child div css appears to be handling it.

```suggestion
```